main/tests/cypress/cypress/integration/tutorial/layout-of-openrefine.spec.js

```diff
@@ -0,0 +1,64 @@
+// This spec is implementation for chapter 03 of following tutorial https://librarycarpentry.org/lc-open-refine/02-importing-data/index.html
```

Comment:
This is probably https://librarycarpentry.org/lc-open-refine/03-working-with-data/index.html

Reply:
Ooops fixing